cmd/config/cmd/shell-complete/shell-complete.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package shell_complete
```

Comment:
can we avoid the snake case and here to be consistent w/ package names?
the underscore in package name has special meaning sometimes (_test)

Reply:
Done